contracts/OpenSTValue.sol

```diff
@@ -223,6 +224,10 @@ contract OpenSTValue is OpsManaged, Hasher {
             hashLock:     _hashLock
         });
 
+        // store the staking intent hash directly in storage of OpenSTValue 
+        // so that a Merkle proof can be generated for active staking intents
+        intents[hashIntentKey(_staker, nonce)] = stakingIntentHash;
```

Comment:
hashIntentKey(_staker, nonce) it can be taken in a variable and should be part of event. Platform will need this to generate storage proof.

Reply:
It should be part of StakingIntentDeclared event.

Reply:
Done!